stacks-core/create-source-binary/build-scripts/Dockerfile.macos-x64

```
@@ -8,6 +8,7 @@ ARG OSXCROSS="https://github.com/hirosystems/docker-osxcross-rust/releases/downl
 ARG TARGET=x86_64-apple-darwin
 ARG TARGET_CPU
 ENV RUSTFLAGS="${TARGET_CPU:+${RUSTFLAGS} -Ctarget-cpu=${TARGET_CPU}}"
+ARG TAG="No Tag Info"
```

```suggestion
ARG TAG
```
this should be enough - specifying a string for the tag isn't necessary if it's not set (i.e. the case will check for `singer-*` only)